### PR TITLE
Initial contribution for Helium

### PR DESCRIPTION
**Pull request by pe1mew**

### CODE REVIEW

File: integration/helium.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+2. Create label and link devices
+3. Configure your TTN Mapper application in **Flows**
+
+### 1. Create the integration
+
+On Helium console, click on the *add (+)* icon in the bottom left and select **Add integrations**. Then click on HTTP. In the configuration page for the integration fill in the following:
+
+* **Endpoint URL (Required)**: Enter `https://integrations.ttnmapper.org/helium/v1`
+* **Step 3 - Name your Integration (Required)**: Enter the name for your integration
+* **Endpoint**: Add `https://integrations.ttnmapper.org/chirp/v3/events`
```
Comment (jpmeijers) [inline integration/helium.md]: Remove chirpstack endpoint

File: integration/helium.md [hunk 2]
```diff
@@ -1 +1,86 @@
...
+* **HTTP Headers (Mandatory)**: Add header **TTNMAPPERORG-NETWORK** and set it to 'HELIUM'
```
Comment (jpmeijers) [inline integration/helium.md:39]: Network name is not required for helium

File: integration/helium.md [hunk 3]
```diff
@@ -1 +1,86 @@
...
+* **HTTP Headers (Optional)**: Add headers to the integration as specified below at **Headers**.
+* Click **Add Integration** to confirm and save your integration.
+
+#### Optional Headers
+
+* **TTNMAPPERORG-EXPERIMENT**: You only need to provide an experiment name if you are testing and do not want your mapping results to contribute to the main coverage map for the network. In other words, when you are mapping normally with a device that is between 1 and 2 metre above ground level, you can leave this field blank. If you are doing anything else like launching and tracking a balloon, please provide a unique experiment name here. Adding the date to the experiment name helps to find it again later. Also see [Experiments](#experiments).
+* **TTNMAPPERORG-USER**: Add your email address in this header to allow TTN Mapper to associate your coverage data with you. This will allow to easily find your data, and also verify when you request your data to be deleted.
```
Comment (jpmeijers) [inline integration/helium.md]: The user is a required header

File: integration/helium.md [hunk 4]
```diff
@@ -1 +1,86 @@
...
 
+A HTTP Integration allows you to upload data to TTN Mapper. 
+
+The goal of TTN Mapper is to provide a map of the actual coverage of LoraWAN Hotspots. Contributors to TTN Mapper measure the performance of Hotspots in their vicinity and upload this information to the TTN Mapper website. Here the information is aggregated and shared with the TTN community.
+
+Go to https://ttnmapper.org for the global coverage map.
+
+
+## Prerequisites
+
```
Comment (jpmeijers) [inline integration/helium.md:11]: We can maybe just say: use the same format as is required by Cargo

File: integration/helium.md (hunk 3 above)
Comment (pe1mew) [reply integration/helium.md]: I will change but chirpstack documentation is not consistent with this statement ;-)

File: integration/helium.md (hunk 4 above)
Comment (pe1mew) [reply integration/helium.md:11]: I will add the suggestion but I find we should provide our own specification here.

File: integration/helium.md (hunk 2 above)
Comment (pe1mew) [reply integration/helium.md:39]: Done

File: integration/helium.md (hunk 3 above)
Comment (pe1mew) [reply integration/helium.md]: Done

**Comment by pe1mew** [comment]
Changes as requested are applied.

File: integration/helium.md [hunk 5]
```diff
@@ -1 +1,87 @@
...
+Once the integration is enabled any message sent by the end device will also be published to the TTN Mapper website. In order for TTN Mapper to correctly interpret the incoming messages, a payload decoder needs to be configured. 
+* If your device is sending data in Cayenne LPP format, choose this decoder. 
+* Otherwise create a custom javascript decoder which decoded the raw payload into one of these formats:
+  1. A JSON object containing the keys "**latitude**", "**longitude**" and "**altitude**". Some variations of these keys will also work. 
+  2. In addition the JSON object should preferably contain one of the following keys "**hdop**", "**accuracy**" or "**sats**".
+
+> As an alternative teh payload decoder for [Helium Cargo](https://docs.helium.com/use-the-network/console/integrations/cargo/) or [Helium Mappers](https://docs.helium.com/use-the-network/coverage-mapping/mappers-quickstart/)mappers can be used as well with TTN Mapper.
```
Comment (jpmeijers) [inline integration/helium.md:20]: teh/the

File: integration/chirpstack.md [hunk 6]
```diff
@@ -30,12 +30,13 @@ On the Chirpstack-UI, open your application and then click on the *Integrations*
 * **Endpoint**: Add `https://integrations.ttnmapper.org/chirp/v3/events`
 
 ### Headers
-#### Required
+
+#### Mandatory
 * **TTNMAPPERORG-NETWORK**: Name of your network, for example: `my.network.name`. Try and make this unique, as this value will be used to distinguish your network's coverage and gateways from other ChirpStack networks' coverage and gateways. This header is a temporary workaround until a proper network identifier is added to ChirpStack. See [this Github issue](https://github.com/brocaar/chirpstack-network-server/issues/532)
+* **TTNMAPPERORG-USER**: Add your email address in this header to allow TTN Mapper to associate your coverage data with you. This will allow to easily find your data, and also verify when you request your data to be deleted.
```
Comment (jpmeijers) [inline integration/chirpstack.md:36]: Please revert this change. Because ChirpStacks are inherently private networks, we do not enforce identifying the data like we do on public networks like TTN an Helium.

File: integration/helium.md (hunk 2 above)
Comment (jpmeijers) [reply integration/helium.md:39]: I think you forgot to remove it.
